docs/user/logs.md

```diff
@@ -0,0 +1,597 @@
+
+```bash
+kubectl get logpipeline
+NAME      CONFIGURATION GENERATED   GATEWAY HEALTHY   AGENT HEALTHY   FLOW HEALTHY
+backend   True                      True              True            True        
+```
+
+## Kyma Modules With Logging Capabilities
+
+Kyma bundles modules that can be involved in user flows. If you want to collect all logs of all modules, enable the `application` input for the `kyma-system` namespace.
+
+### Istio
+
+The Istio module is crucial as it provides the [Ingress Gateway](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/). Typically, this is where external requests enter the cluster scope. Furthermore, every component that’s part of the Istio Service Mesh runs an Istio proxy. Using the Istio telemetry API you can enable access logs for the Ingress Gateway and the proxies individually.
+
+The Istio module is configured with an [extension provider](https://istio.io/latest/docs/tasks/observability/telemetry/) called `kyma-logs`. To activate the provider on the global mesh level using the Istio [Telemetry API](https://istio.io/latest/docs/reference/config/telemetry), place a resource to the `istio-system` namespace. The following code samples help setting up the Istio logging feature:
+
+The following example configures all Istio proxies with the `kyma-logs` extension provider, which, by default, reports access logsto the log gateway of the Telemetry module.
+
+```yaml
+apiVersion: telemetry.istio.io/v1
+kind: Telemetry
+metadata:
+  name: mesh-default
+  namespace: istio-system
+spec:
+  tracing:
+  - providers:
+    - name: "kyma-logs"
+```
+
+## Operations
+
+A LogPipeline runs several OTel Collector instances in your cluster. This Deployment serves OTLP endpoints and ships received data to the configured backend.
+
+The Telemetry module ensures that the OTel Collector instances are operational and healthy at any time, for example, with buffering and retries. However, there may be situations when the instances drop logs, or cannot handle the log load.
+
+To detect and fix such situations, check the [pipeline status](./resources/02-logpipeline.md#logpipeline-status) and check out [Troubleshooting](#troubleshooting). If you have set up [pipeline health monitoring](./04-metrics.md#5-monitor-pipeline-health), check the alerts and reports in an integrated backend like [SAP Cloud Logging](./integration/sap-cloud-logging/README.md#use-sap-cloud-logging-alerts).
+
+> [! WARNING]
+> It's not recommended to access the metrics endpoint of the used OTel Collector instances directly, because the exposed metrics are no official API of the Kyma Telemetry module. Breaking changes can happen if the underlying OTel Collector version introduces such.
+> Instead, use the [pipeline status](./resources/05-metricpipeline.md#metricpipeline-status).
+
+## Limitations
+
+- **Throughput**: The maximum throughput is limited. The actual number gets defined when the feature is out of experimental status.
```

Comment:
```suggestion
- **Throughput**: The maximum throughput is limited.
```
I don't think we need this sentence right now. Let's add it when we know the specific number.

Reply:
done